assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * Internal dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { useAPI } from '../../app/api';
+import { useConfig } from '../config';
+
+function useUploader() {
+	const { actions: { uploadMedia } } = useAPI();
+	const { storyId, maxUpload, allowedMimeTypes: { image: allowedImageMimeTypes, video: allowedVideoMimeTypes } } = useConfig();
+	const allowedMimeTypes = [ ...allowedImageMimeTypes, ...allowedVideoMimeTypes ];
+
+	const isValidType = useCallback( ( { type } ) => {
+		return allowedMimeTypes.includes( type );
+	}, [ allowedMimeTypes ] );
+
+	const fileSizeCheck = useCallback( ( { size } ) => {
+		return size < maxUpload;
```

Comment:
Should this be `<=`?

Reply:
Consider this value is measured in bytes, allowing one more byte seems a little pointless here. Isn't it safer to just make sure that file is below the limit and not exact it?

Reply:
It's just more accurate with `<=` since the `maxUpload` exact value should also be allowed, not a big difference indeed.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/195/commits/6f96c2da6a7a9e87505cc7ccbffb07b668079405